receiver/carbonreceiver/factory.go

```diff
@@ -13,7 +14,11 @@ import (
 
 	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/carbonreceiver/internal/metadata"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/carbonreceiver/protocol"
-	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/carbonreceiver/transport"
+)
+
+const (
+	// TCPIdleTimeoutDefault is the default timeout for idle TCP connections.
+	TCPIdleTimeoutDefault = 30 * time.Second
```

Comment:
Do we need to export this? Maybe just copy it to the wavefront receiver instead?

Reply:
Done.